.drone.yml

```
@@ -2937,6 +3081,294 @@ steps:
+      - ssh-keyscan -H github.com > $WORKSPACE_DIR/.ssh/known_hosts 2>/dev/null
+      - chmod 600 $WORKSPACE_DIR/.ssh/known_hosts
+      - GIT_SSH_COMMAND='ssh -i $WORKSPACE_DIR/.ssh/id_rsa -o UserKnownHostsFile=$WORKSPACE_DIR/.ssh/known_hosts -F /dev/null' git submodule update --init e
+      # this is allowed to fail because pre-4.3 Teleport versions don't use the webassets submodule
+      - GIT_SSH_COMMAND='ssh -i $WORKSPACE_DIR/.ssh/id_rsa -o UserKnownHostsFile=$WORKSPACE_DIR/.ssh/known_hosts -F /dev/null' git submodule update --init --recursive webassets || true
+      - rm -rf $WORKSPACE_DIR/.ssh
+      - mkdir -p $WORKSPACE_DIR/go/artifacts $WORKSPACE_DIR/go/cache
+      - echo "${DRONE_TAG##v}" > $WORKSPACE_DIR/go/.version.txt
+      - cat $WORKSPACE_DIR/go/.version.txt
+      - echo "${BORINGCRYPTO_BRANCH}" > $WORKSPACE_DIR/go/.bc_commit.txt
+      - cat $WORKSPACE_DIR/go/.bc_commit.txt
+
+  - name: Build the boringcrypto Go builder
+    environment:
+      WORKSPACE_DIR: /tmp/build-darwin-amd64-fips
+      GODL_DIR: /tmp/build-darwin-amd64-fips/go/src/github.com/gravitational/godl
+    commands:
+      - set -u
+      - cd $GODL_DIR
+      - mkdir build
+      - export BORINGCRYPTO_BRANCH=$(cat $WORKSPACE_DIR/go/.bc_commit.txt)
+      - go build -o $GODL_DIR/build/$BORINGCRYPTO_BRANCH $BORINGCRYPTO_BRANCH/main.go
+
+  - name: Pull boringcrypto build tarball from S3
+    environment:
+      AWS_S3_BUCKET:
+        from_secret: AWS_S3_BUCKET
+      AWS_ACCESS_KEY_ID:
+        from_secret: AWS_ACCESS_KEY_ID
+      AWS_SECRET_ACCESS_KEY:
+        from_secret: AWS_SECRET_ACCESS_KEY
+      AWS_REGION: us-west-2
+      WORKSPACE_DIR: /tmp/build-darwin-amd64-fips
+      GODL_DIR: /tmp/build-darwin-amd64-fips/go/src/github.com/gravitational/godl
+    commands:
+      - set -u
+      - export BORINGCRYPTO_BRANCH=$(cat $WORKSPACE_DIR/go/.bc_commit.txt)
+      - mkdir -p $HOME/sdk/dev.boringcrypto.go1.15
+      - cd $HOME/sdk/dev.boringcrypto.go1.15
+      - aws s3 cp s3://$AWS_S3_BUCKET/ci/go/$BORINGCRYPTO_BRANCH-darwin-amd64.tar.gz .
+      - aws s3 cp s3://$AWS_S3_BUCKET/ci/go/$BORINGCRYPTO_BRANCH-darwin-amd64.tar.gz.sha256 .
+      - $GODL_DIR/build/$BORINGCRYPTO_BRANCH download
+
+  - name: Build Mac release artifacts
+    environment:
+      GOPATH: /tmp/build-darwin-amd64-fips/go
+      GOCACHE: /tmp/build-darwin-amd64-fips/go/cache
+      # use custom Go compiler
+      GODL_DIR: /tmp/build-darwin-amd64-fips/go/src/github.com/gravitational/godl
+      OS: darwin
+      ARCH: amd64
+      WORKSPACE_DIR: /tmp/build-darwin-amd64-fips
+      FIPS: "yes"
+    commands:
+      - set -u
+      - cd $WORKSPACE_DIR/go/src/github.com/gravitational/teleport
+      - export VERSION=$(cat $WORKSPACE_DIR/go/.version.txt)
+      - export BORINGCRYPTO_BRANCH=$(cat $WORKSPACE_DIR/go/.bc_commit.txt)
+      - make -C e clean release VERSION=$VERSION OS=$OS ARCH=$ARCH FIPS=$FIPS GO_BINARY="$GODL_DIR/build/$BORINGCRYPTO_BRANCH"
```

I think `GITTAG` needs to be set here

```suggestion
      - make -C e clean release GITTAG=v$VERSION VERSION=$VERSION OS=$OS ARCH=$ARCH FIPS=$FIPS GO_BINARY="$GODL_DIR/build/$BORINGCRYPTO_BRANCH"
```